### DISCUSSION TITLE
New to GrapesJS and have some questions

### DISCUSSION THREAD

**Discussion by davepruitt** [GrapesJS/grapesjs#6010] [General]
Hey all,

I am exploring possibly using GrapesJS in a project of mine, and I have some questions. I've walked through the initial "Getting Started" tutorial, so I am familiar with the very basics.

I'd like to make an editor that uses drag-and-drop, and allows the user to drag components to any (x, y) position on the page. As an example, in many applications (whether it be Figma, Powerpoint, etc) when you select an object you frequently see 4 "handles" on each corner of an object that allows you to scale/resize the object, or you can click and drag on the object itself to move it around the canvas/page.

So far in all of my testing, as well as in the searching that I've done of various GrapesJS things, I don't see anything like this. For example, when I click an object on my GrapesJS canvas, the object becomes outlined and has some action buttons on the top right corner. One of those buttons (sometimes) allows moving the object to another place on the page, but it's not _free_ movement.

Maybe I'm not totally grasping things yet, but is there a way to do something like what I am describing in GrapesJS?

**Comment by RakulAgn** [comment]
try `dragMode: 'absolute'` add this to your init and try to move the component as you like 
and for resizing the component you can define the component `resizable:true` on your component definition to enable the resize. @davepruitt

**Reply by davepruitt** [reply]
Thank you! This was very helpful!
